src/index.html

```diff
@@ -20,6 +20,8 @@
   <script src="https://kit.fontawesome.com/061912f229.js" crossorigin="anonymous"></script>
   <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500&display=swap" rel="stylesheet">
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <!-- Scroll to Top button-->
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.0/css/all.css" integrity="sha384-lZN37f5QGtY3VHgisS14W3ExzMWZxybE1SJSEsQp9S+oqd12jhcu+A56Ebc1zFSJ" crossorigin="anonymous">
```

Comment:
Hey @neha-kothari ,
You need not include font awesome. You could rather use bootstrap icons

Thanks

Reply:
Okay! Deleted that.